### PR TITLE
218 lup reviewed logic

### PR DESCRIPTION
**Pull request by hannahkates**
Changes:
- Updated LUP SQL query with revised logic for which projects are assigned `"tab"='reviewed'`. Now it includes projects where the LUP's review milestone is still in progress but the disposition for the LUP's role has been submitted.

I tested this revised logic by...
1. Getting the count of `reviewed` projects currently provided by the API for BKBP (`'2a231d14-693e-e811-8133-1458d04d06c0'`)
2. Picking one of those projects and changing the BP review milestone `statuscode` to "In Progress" instead of "Completed", then checked to see that the `reviewed` response count decreased by one
3. For the same project in step 2, I then filled in data for `dcp_boroughpresidentrecommendation` which was previously blank. I then reran the query and saw that the count of `reviewed` projects again matched the original count from step 1.

Addresses https://github.com/NYCPlanning/labs-zap-api/issues/218

### CODE REVIEW

File: queries/projects/lup-projects.sql [hunk 1]
```diff
@@ -25,7 +31,11 @@ WITH lups_project_assignments AS (
   INNER JOIN -- inner because not all projects should be visible to users based on visibility field in "where" clause
     dcp_project AS p ON lup.dcp_project = p.dcp_projectid
   INNER JOIN -- inner because we only want certain milestones with the status included in the "where" clause
-    dcp_projectmilestone AS mm ON lup.dcp_project = mm.dcp_project
+    (SELECT * FROM dcp_projectmilestone WHERE statuscode <> 'Overridden') AS mm ON lup.dcp_project = mm.dcp_project
+  LEFT JOIN
+    dcp_communityboarddisposition AS disp
+    ON disp.dcp_project = lup.dcp_project
+    AND disp.dcp_recommendationsubmittedby = '${id:value}' -- plugs in contactid
```
Comment (godfreyyeung) [inline queries/projects/lup-projects.sql]: What do lines 35-38 do? Just asking to learn/understand

Comment (hannahkates) [reply queries/projects/lup-projects.sql]: I had to join the `dcp_communityboarddisposition` table onto the LUP project assignment results in order to find out if the LUP had submitted their recommendation yet (see lines 8-10). In order to join the dispositions onto the LUP project assignment, I needed to join using the project id and contact id. Now that I'm looking at this again though, I think I may also need to include `dcp_lupteammemberrole` and the corresponding "representing" field in the disposition record to the join criteria, so we don't accidentally think a BB review is done when the BP has only submitted the BP review not BB yet.

Comment (hannahkates) [reply queries/projects/lup-projects.sql]: @godfreyyeung updated now FYI

File: queries/projects/lup-projects.sql [hunk 2]
```diff
@@ -25,7 +31,12 @@ WITH lups_project_assignments AS (
   INNER JOIN -- inner because not all projects should be visible to users based on visibility field in "where" clause
     dcp_project AS p ON lup.dcp_project = p.dcp_projectid
   INNER JOIN -- inner because we only want certain milestones with the status included in the "where" clause
-    dcp_projectmilestone AS mm ON lup.dcp_project = mm.dcp_project
+    (SELECT * FROM dcp_projectmilestone WHERE statuscode <> 'Overridden') AS mm ON lup.dcp_project = mm.dcp_project
+  LEFT JOIN
+    dcp_communityboarddisposition AS disp
```
Comment (allthesignals) [inline queries/projects/lup-projects.sql]: hmm why does this need to be aliased to something other than its original name?

File: queries/projects/lup-projects.sql [hunk 3]
```diff
@@ -86,15 +97,15 @@ SELECT
...
         'vote_date', disp.dcp_dateofvote,
         'vote_location', disp.dcp_votelocation,
-        -- 'vote_infavor', disp.dcp_votinginfavorrecommendation,
-        -- 'vote_against', disp.dcp_votingagainstrecommendation,
-        -- 'vote_abstained', disp.dcp_votingabstainingonrecommendation,
-        -- 'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
+        'vote_infavor', disp.dcp_votinginfavorrecommendation,
+        'vote_against', disp.dcp_votingagainstrecommendation,
+        'vote_abstained', disp.dcp_votingabstainingonrecommendation,
+        'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
```
Comment (allthesignals) [inline queries/projects/lup-projects.sql]: these should be the same as the original column names so things are bit more predictable in the frontend

File: queries/projects/lup-projects.sql [hunk 4]
```diff
@@ -86,15 +97,15 @@ SELECT
...
         'vote_quorum', disp.dcp_wasaquorumpresent,
         'rec_bb', disp.dcp_boroughboardrecommendation,
         'rec_cb', disp.dcp_communityboardrecommendation,
         'rec_bp', disp.dcp_boroughpresidentrecommendation
-        -- 'rec_comment', disp.dcp_consideration
+        'rec_comment', disp.dcp_consideration
```
Comment (allthesignals) [inline queries/projects/lup-projects.sql]: i know these were uncommented and were existing there before but we should keep the column names the same

**Comment by allthesignals** [comment]
will be merged manually

**Pull request closed without merging**